code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -216,3 +216,9 @@
 		B.deity_name = "Success"
 		B.icon_state = "greentext"
 		B.item_state = "greentext"
+
+/obj/item/weapon/storage/box/syndie_kit/cutouts/New()
+	..()
+	for(var/i in 1 to 3)
+		new/obj/item/cardboard_cutout/adaptive(src)
+	new/obj/item/toy/crayon/rainbow(src)
```

Comment:
I think a spraycan would probably be more appropriate here, and you need to declare a name for this typepath